tests/python/unittest/test_sparse_ndarray.py

```diff
@@ -266,12 +285,109 @@ def test_sparse_nd_negate():
     # we compute (-arr)
     assert_almost_equal(npy, arr.asnumpy())
 
+
 def test_sparse_nd_output_fallback():
     shape = (10, 10)
     out = mx.sparse_nd.zeros('row_sparse', shape)
     mx.nd.random_normal(shape=shape, out=out)
     assert(np.sum(out.asnumpy()) != 0)
 
+
+def test_sparse_ndarray_pickle():
```

Comment:
We should also explicitly test module.save / load in test_module.py, after we refactor all the NDArray(() constructor